src/components/Api.js

```diff
@@ -0,0 +1,86 @@
+export default class Api {
+  constructor(options) {
+    this._baseUrl = options.baseUrl;
+    this._headers = options.headers;
+  }
+
+  _checkResponse(res) {
+    if (res.ok) {
+      return res.json();
+    } else {
+    return Promise.reject(`Error: ${res.status}`);
```

Comment:
Needs correction: It's important to format code correctly, use the indentation hierarchy, and split your code. Please check these sources to study the standard: [https://code.s3.yandex.net/web-developer/landings/design-rules-en/index.html](https://code.s3.yandex.net/web-developer/landings/design-rules-en/index.html), [https://javascript.info/coding-style](https://javascript.info/coding-style).

To make sure your code is formatted correctly, we suggest using Prettier (https://prettier.io/). It can be integrated with any code editor or IDE.

You should fix the formatting in all the files